core/templates/dev/head/player/conversation_skin_directive.html

```diff
@@ -154,10 +154,8 @@
           </md-button>
         </div>
 
-        <div ng-if="!exploration.isInteractionInline(activeCard.stateName)">
```

Comment:
I think it can be deleted. We had a design sometime back where we allowed things other than interactions in the supplemental card, but that idea was scrapped.